sycl/test/basic_tests/accessor/host_acc_opt.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clangxx -O2 -fsycl -S -emit-llvm %s -o - | FileCheck %s
```

Comment:
I suppose this test is supposed to call only a host compiler.

Reply:
It there a convenient way to call the host compiler only?

Reply:
I think `-fsycl-host-only` might be helpful.

Reply:
https://github.com/intel/llvm/blob/sycl/sycl/test/lit.cfg.py#L89 - it looks like it's a front-end option.